### PR TITLE
Add guide "Connecting to Internet With ESP8266-based MCUs"

### PR DESCRIPTION
**Pull request by evgenykochetkov**

### CODE REVIEW

File: docs/guide/esp8266-native-wifi/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Connecting to Internet With ESP8266-based MCUs
+---
```
Comment (knopki) [inline docs/guide/esp8266-native-wifi/README.md]: Please, add version, description, image (if any).

File: docs/guide/http-get/README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Fetching Data from Web API’s with HTTP GET requests
+version: 1.0.1
```
Comment (knopki) [inline docs/guide/http-get/README.md]: Please, add description, image (if any).

File: docs/guide/esp8266-native-wifi/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+[`xod-dev/esp8266-mcu/lan-ip`](https://xod.io/libs/xod-dev/esp8266-mcu/lan-ip/) gets
+the IP address from `INET` and
+[`xod/net/format-ip`](https://xod.io/libs/xod/net/format-ip/) 
+formats the IP as a human-readable string like “192.168.88.101”.
+
+To output the progress to the debugger we use `select` and `console-log`. Bind `"Connecting"` to `select`'s `X1`, `On Boot` to `S1` and `"Error"` to `X3`. 
+
+Upload the patch to your board in debug mode. You should see something like this:
+
+![observing result in debugger](./result-in-debugger.png)
```
Comment (nkrkv) [inline docs/guide/esp8266-native-wifi/README.md]: A “what’s next” section is desired. Pointing `docs/guide/http-get` is a good idea for its content.

File: docs/guide/esp8266-native-wifi/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# Connecting to Internet With ESP8266-based MCUs
+
+ESP8266-based boards such as NodeMCU and WeMos D1 Mini are quite popular for projects that require internet connection because of their built-in WiFi capabilities. 
+
+XOD provides [`xod-dev/esp8266-mcu`](https://xod.io/libs/xod-dev/esp8266-mcu/) library that makes connecting to the internet very simple. All you need is a [`connect`](https://xod.io/libs/xod-dev/esp8266-mcu/connect/) node:
+
+![connect node](./1-connect-node.patch.png)
+
+Bind your network SSID and password to `SSID` and `PWD` pins and you are good to go. Link `INET` to nodes that need an established
+internet connection (for example, to [make HTTP requests](../http-get/)).
```
Comment (nkrkv) [inline docs/guide/esp8266-native-wifi/README.md]: Suggest moving the last sentence to the end of the article

File: docs/guide/esp8266-native-wifi/README.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+title: Connecting to Internet With ESP8266-based MCUs
+---
+
+# Connecting to Internet With ESP8266-based MCUs
+
+ESP8266-based boards such as NodeMCU and WeMos D1 Mini are quite popular for projects that require internet connection because of their built-in WiFi capabilities. 
+
+XOD provides [`xod-dev/esp8266-mcu`](https://xod.io/libs/xod-dev/esp8266-mcu/) library that makes connecting to the internet very simple. All you need is a [`connect`](https://xod.io/libs/xod-dev/esp8266-mcu/connect/) node:
```
Comment (nkrkv) [inline docs/guide/esp8266-native-wifi/README.md]: “makes very simple” is opinionated. Suggest removing “very.“

File: docs/guide/http-get/README.md (hunk 2 above)
Comment (nkrkv) [reply docs/guide/http-get/README.md]: Not really related to this PR.

File: docs/guide/esp8266-native-wifi/README.md (hunk 1 above)
Comment (nkrkv) [reply docs/guide/esp8266-native-wifi/README.md]: +1

For the image, I would suggest taking a photo of NodeMCU and WeMos together.

**Comment by evgenykochetkov** [comment]
Updated. Check again, please

**Pull request merged**